src/core/components/textComponents/TextComponents.styled.ts

```diff
@@ -0,0 +1,39 @@
+import {fontFamilies, Sizes, fontSizes, FontType, lineHeights, indents} from "@constants/Typography";
+import styled, {DefaultTheme} from "styled-components/native";
+import {Colors} from "@constants/Colors";
+
+export interface StyledCustomText {
+    theme: DefaultTheme,
+    size: Sizes;
+    fontType: FontType;
+    fontColor: Colors;
+    align: 'start' | 'center' | 'end',
+    marginLeft: typeof indents,
+    marginRight: typeof indents,
+    marginTop: typeof indents,
+    marginBottom: typeof indents,
+}
+
+export const StyledCustomText = styled.Text((props: StyledCustomText) => ({
+    color: props.theme.colors[props.fontColor], // change to fontColor
+    fontSize: fontSizes[props.size],
+    lineHeight: lineHeights[props.size],
+    fontFamily: fontFamilies[props.fontType],
+    textAlign: props.align
+}));
+
+export const IconWrapper = styled.View.attrs<StyledCustomText>(() => ({
+    marginTop: '0',
+    marginBottom: '0',
+    marginLeft: '0',
+    marginRight: '0'
```

Comment:
margin: 0?

Reply:
Ah, it's attribute? Why do you have attributes for every margin for this element? Just make some element with hardcoded margin and use it.

Reply:
Thanks a lot. :) Most of my usual questions not "how to do it?" but "how to do it RIGHT WAY?". So I really appreciate  your efforts to helpi me understand how to do it ideologically correct way!